README.md

```diff
@@ -116,12 +113,9 @@ If you were already logged in, log out and log back in, then you'll see an "admi
 
 This command will start both the frontend and the api together
 
-     $ yarn start 
-
-To start the services individually do:
+     $ yarn run dev
 
-     $ yarn api
-     $ yarn frontend
+To start the services individually, execute the above command in the relevant directory (either `api` or `frontend`)
```

Comment:
This doesn't seem to work; what will?

Reply:
These aren't really run separately after the migration to next.js. I'll push a commit that fixes this.